pyproject.toml

```diff
@@ -20,6 +20,7 @@ pydantic = "^1.9.1"
 SQLAlchemy = "^1.4.39"
 ovld = "^0.3.2"
 questionary = "^1.10.0"
+openreview-py = {git = "https://github.com/openreview/openreview-py.git"}
```

Comment:
Is there a reason why it is installed from git? `pip install openreview-py` seems to work well enough.

Reply:
I just followed what they suggest in their docs (https://docs.openreview.net/getting-started/using-the-api/installing-and-instantiating-the-python-client) but will check if the pypi package looks up to date

Reply:
pypi package is not up to date with master but it is with the latest version tag which is probably better